utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -196,6 +208,18 @@ if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
 	fi
 fi
 
+if [[ ! -f "$DAOS_INSTALL_DIR/bin/activate" ]] ; then
+	info "Installing avocado"
+	run $PYTHON_EXE -m venv "$DAOS_INSTALL_DIR"
+	source "$DAOS_INSTALL_DIR/bin/activate"
+	run $PIP_EXE install --upgrade pip
+	run $PIP_EXE install -r "$DAOS_SOURCE_DIR/requirements.txt"
+	deactivate
+	info "Customize venv for DAOS install"
+	run rm -fv "$DAOS_INSTALL_DIR/lib64"
+	run mkdir -pv "$DAOS_INSTALL_DIR/lib64"
+fi
+
 cd "$DAOS_SOURCE_DIR"
 SCONS_OPTS="-j $JOBS_NB --implicit-deps-changed"
 case $CMD in
```

Comment:
`--config-force` is an option that seems to be used ubiquitously but not actually required.  If you can demonstrate that it's required then please file a ticket but in practice it shouldn't be needed here.  The same probably applies to the `--implicit-deps-changed` option.

There's a `--build-deps=only` option, the install could call this and then fall through to the update case which would perhaps be clearer.

Reply:
I added these two options, as they were at required at a point of time.
I will check, if they are still needed for my use case.
If no, I will remove them.

- [ ] Check if --config-force and --implicit-deps-changed options are still needed.
- [ ] Build in two steps with --build-deps=only option.

Reply:
> I added these two options, as they were at required at a point of time. I will check, if they are still needed for my use case. If no, I will remove them.
> 
> * [x]  Remove useless scons option --config-force and --implicit-deps-changed.

Fixed with commit f8c49965c035fd3a39fe24c51e09a44e628bf0d4